src/main/java/nl/openvalue/CustomerStatementProcessor/validator/impl/ValidatorFacade.java

```diff
@@ -0,0 +1,48 @@
+package nl.openvalue.CustomerStatementProcessor.validator.impl;
+
+import nl.openvalue.CustomerStatementProcessor.model.Statement;
+import nl.openvalue.CustomerStatementProcessor.util.ErrorFileHelper;
+import nl.openvalue.CustomerStatementProcessor.validator.api.ValidatorService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.math.BigDecimal;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+@Service
+public class ValidatorFacade implements ValidatorService {
```

Comment:
Ik denk dat je nog wat validatie mist, want waar kijk je niet echt of Transaction reference numeriek is, of Account number een IBAN is, Start balance wel echt een BigDecimal is, etc.
Je logt wel als 't niet geparsed kan worden, maar volgens mij worden die niet door de `ErrorFileHelper` afgehandeld.